Kickstarter-iOS/Views/Controllers/SheetOverlayViewController.swift

```diff
@@ -80,11 +90,16 @@ final class SheetOverlayViewController: UIViewController {
     NSLayoutConstraint.activate([
       childView.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
       childView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor),
-      childView.widthAnchor.constraint(equalToConstant: portraitWidth)
+      childView.widthAnchor.constraint(equalToConstant: portraitWidth),
+      childView.topAnchor
+        .constraint(
+          greaterThanOrEqualTo: self.view.topAnchor,
```

Comment:
does this mean that the child view's top anchor can technically be higher than the overlay view it's inside?

Reply:
Greater than or equal to, in this case, implies that the childs top anchor will be "lower" on the screen than the parent. As the offset increases in number, the top anchor gets lower in the frame.